<launch>
    <!-- Start the fake_imu_publisher node with parameters defined within the node scope -->
    <node name="fake_imu_publisher" pkg="tf_to_fake_imu" type="fake_imu_publisher_node" output="screen">
        <param name="imu_topic_name_out" value="/fake_imu" />
        <param name="tf_world_frame_id" value="map" />
        <param name="tf_mobile_frame_id" value="base_link" />
        <param name="gravity" value="[0.0, 0.0, -9.81]" />
        <param name="rate" value="100" />
    </node>
</launch>